src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -2,35 +2,74 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Runtime.InteropServices;
+
+using size_t = System.UInt64;
```

Comment:
This is fine for now.  We are going to need to put this stuff into a native shim (@nguerrera's work) to deal with differences in size for when we start looking at platforms where this isn't the case.